script/policy_source_helper.py

```diff
@@ -16,8 +16,13 @@ def AddBravePolicies(template_file_contents):
             'name': 'TorDisabled',
             'type': 'main',
             'schema': {'type': 'boolean'},
-            'supported_on': ['chrome.win:78-'],
-            'features': {'dynamic_refresh': False, 'per_profile': False},
+            'supported_on': ['chrome.*:78-'],
```

Comment:
I think this should be `['chrome.win:78-', 'chrome.*:93-']`

Reply:
Yup, it's accurate

Reply:
Updated.